pandas/tests/frame/methods/test_transpose.py

```diff
@@ -63,7 +65,7 @@ def test_transpose_tzaware_2col_mixed_tz(self):
         df4 = DataFrame({"A": dti, "B": dti2})
         assert (df4.dtypes == [dti.dtype, dti2.dtype]).all()
         assert (df4.T.dtypes == object).all()
-        tm.assert_frame_equal(df4.T.T, df4)
+        tm.assert_frame_equal(df4.T.T, df4, check_dtype=False)
```

Comment:
can you cast the expected here to keep the test strict

Reply:
done